### ISSUE TITLE
Can't load Clojure Bundle into TextMate 2

### ISSUE THREAD

**Issue by Lynchnco** [swannodette/textmate-clojure#37]
I was able to get the first two commands to work but the third command fails - see following:

> osascript -e 'tell app "TextMate" to reload bundles'
> 23:37: syntax error: A identifier can’t go after this identifier. (-2740)

I'm a complete newbie to Clojure, so be kind. 
I don't know what version of Clojure that I'm using but it could be 1.5.1.

I'd appreciate any help

**Comment by ghost** [comment]
I'm having the same issue. Did you find a solution?

**Comment by Lynchnco** [comment]
Hi,
…I gave up on Clojure - I just don’t like the language.
…I’m sticking with Haskell and F# and will take a look at Scala.
…I was able to get Clojure in TextMate1 but gave up on TextMate2 and went to TextMate3. I didn’t try loading the bundle into TextMate3.
…I contacted Michael Sheets tm-support@macromates.com in order to get Haskell bundle into TextMate3 - his advice worked. Why don’t you try contacting him [he is a TextMate tech].

…I’m trying, quite unsuccessfully trying to get a “handle” on Category Theory but it’s proving to be difficult.

Good luck to both of us,
Patrick
